docs/cloud/managing-airbyte-cloud.md

```diff
@@ -85,22 +85,27 @@ To switch between workspaces:
 3. Click the name of the workspace you want to switch to.
 
 ### Choose your default data residency
-Default data residency allows you to choose where your data is processed. When you set the default data residency, it is applied to all new connections, but it does not affect existing connections. 
-
-For individual connections, you can choose a data residency that is different from the default. You can do this in the [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection).
+Default data residency allows you to choose where your data is processed.
 
 :::note 
 
-Your data is processed in the chosen residency, but some data is still stored in the US.
+Configuring default data residency only applies it to new connections and does not affect existing connections.   
 
 :::
 
+For individual connections, you can choose a data residency that is different from the default through [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection).
+
+:::note 
+
+While the data is processed in a data plane in the chosen residency, the cursor and primary key data is stored in the US control plane. If you have data that cannot be stored in the US, do not use it as a cursor or primary key.
```

Comment:
@Amruta-Ranade I made some changes to the wording in this section. Let me know if you have suggestions for wording.

Reply:
Looks good to me!